target/src/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "antlr4ts",
-  "version": "0.4.0-dev",
+  "version": "0.4.1-alpha.0",
```

Comment:
❗️ This should not change here. Also, there are several consistency problems with it:

1. The package numbers should always end with `-dev` except for tagged (released) individual commits.
2. All package version numbers should match, and they should all be updated together.
3. No other changes should appear in the commit that changes the version number.
4. The commit immediately following a version number change for release should restore the `-dev` suffix.

See 5b6a5b294c49dfcd951fdb028d19b836e563f8f8 followed immediately by 75340dc212b30027f9e330d78bce544d564033f5 for an example.